tests/integration/test_run.py

```
@@ -202,3 +202,61 @@ def test_run_mixed_compilation(self, model_name: str, model_dir: Path, slices_ou
         assert "slice_4: ezkl" in out
         # onnx for 1
         assert "slice_1: onnx" in out
+
+    @pytest.mark.parametrize("model_name", ["doom"])
+    def test_run_with_tiling(self, model_name: str, model_dir: Path, slices_output_dir: Path, capfd, jstprove_available):
```

Ruff (0.14.11)

Unused method argument: `model_name`

(ARG002)